src/Sylius/Bundle/CoreBundle/MessageDispatcher/ResendOrderConfirmationEmailDispatcher.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\MessageDispatcher;
+
+use Sylius\Bundle\CoreBundle\Message\Admin\ResendOrderConfirmationEmail;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class ResendOrderConfirmationEmailDispatcher implements ResendOrderConfirmationEmailDispatcherInterface
+{
+    public function __construct(private MessageBusInterface $messageBus)
+    {
+    }
+
+    public function dispatch(string $orderToken): void
```

Comment:
Same as in parallel order shipment confirmation PR
https://github.com/Sylius/Sylius/pull/15715#discussion_r1447809855
https://github.com/Sylius/Sylius/pull/15715#discussion_r1448837020